### PR TITLE
Add io.github.swordpuffin.splices

### PR DESCRIPTION
**Pull request by SwordPuffin**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
  A simple game where you make words based on a randomized set of letters. Made with GTK4 and Python.
  See: https://github.com/SwordPuffin/Splices

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. 

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: io.github.swordpuffin.splices.desktop
Comment (hfiguiere) [inline io.github.swordpuffin.splices.desktop:1]: this is never referenced

File: io.github.swordpuffin.splices.metainfo.xml
Comment (hfiguiere) [inline io.github.swordpuffin.splices.metainfo.xml:1]: this is never referenced

File: io.github.swordpuffin.splices.svg
Comment (hfiguiere) [inline io.github.swordpuffin.splices.svg:1]: this is never referenced

File: io.github.swordpuffin.splices.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	<description>
+	  <p>A small time killer where you make words out of randomly generated letters. You can play in either free play mode, free play with a timer, or normal mode where the goal is maximize the length of the words</p>
+	</description>
+  <content_rating type="oars-1.1" />
+  <url type="homepage">https://github.com/SwordPuffin/Splices</url>
+  <releases>
+    <release version="1.0.0" date="2024-10-17">
+      <description>
+        <url> type="details">https://github.com/SwordPuffin/Splices/releases/tag/v1.0.0</url>
+      </description>
```
Comment (hfiguiere) [inline io.github.swordpuffin.splices.metainfo.xml:25]:
```suggestion
        <url type="details">https://github.com/SwordPuffin/Splices/releases/tag/v1.0.0</url>
```

this wasn't even valid XML

**Comment by bbhtt** [comment]
You already have icons, desktop file and metainfo templates hooked to meson upstream, remove them from here and use those

**Comment by bbhtt** [comment]
bot, build io.github.swordpuffin.splices

**Comment by bbhtt** [comment]
can you fix the validation errors? Search for the error code in the link.

```json
{
    "errors": [
        "appstream-failed-validation",
        "appstream-missing-developer-name"
    ],
    "warnings": [
        "appstream-screenshot-missing-caption"
    ],
    "appstream": [
        "E: io.github.swordpuffin.splices:~: desktop-app-launchable-missing"
    ],
    "info": [
        "appstream-missing-developer-name: No developer tag found in Metainfo file",
        "appstream-screenshot-missing-caption: One or more screenshots are missing captions in the Metainfo file",
        "appstream-failed-validation: Metainfo file /tmp/tmpegfweo05/metainfo/io.github.swordpuffin.splices.metainfo.xml has failed validation. Please see the errors in appstream block"
    ],
    "message": "Please consult the documentation at https://docs.flathub.org/docs/for-app-authors/linter"
}
```

**Comment by SwordPuffin** [comment]
I added the developer id and launchable tag to the metainfo.xml file. You can try again.

**Comment by bbhtt** [comment]
bot, build io.github.swordpuffin.splices

**Comment by bbhtt** [comment]
did you update the tag/commit?

**Comment by SwordPuffin** [comment]
I updated the json file to the most recent release which contains the fixes.

Sorry about not noticing it before.

**Comment by bbhtt** [comment]
bot, build io.github.swordpuffin.splices

**Comment by bbhtt** [comment]
bot, build io.github.swordpuffin.splices

**Pull request closed without merging**